util/jsonrpc-types/src/uint64.rs

```diff
@@ -0,0 +1,136 @@
+use ckb_types::prelude::{Pack, Unpack};
+use ckb_types::{core, packed};
+use serde::de::{Error, Visitor};
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+use std::fmt;
+
+#[derive(Clone, Default, PartialEq, Eq, Hash, Debug)]
+pub struct Uint64(u64);
```

Comment:
Why not derive `Copy`？

Reply:
Oh, I didn't think about it. I will update it later.